a_star_planner:
  ros__parameters:
    map_frame: map
    base_frame: base_link
    displacement_threshold: 5.0
    lethal_cost_threshold: 30.0
